pyproject.toml

```diff
@@ -1,18 +1,20 @@
-# Copyright (c) 2023, NVIDIA CORPORATION.
+# Copyright (c) 2023-2025, NVIDIA CORPORATION.
 
 [build-system]
 requires = [
     "hatch-jupyter-builder>=0.5",
     "hatch-nodejs-version>=0.3.2",
-    "hatchling>=1.5.0",
+    "hatchling>=1.27.0",
     "jupyterlab>=4.0.0",
 ] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit dependencies.yaml and run `rapids-dependency-file-generator`.
 build-backend = "hatchling.build"
 
 [project]
 name = "jupyterlab_nvdashboard"
 readme = "README.md"
-license = { file = "LICENSE" }
+license-files = [
+    "LICENSE",
+]
```

Comment:
We drop the `license` field as it now is pulled in from `package.json`

Also [PEP 639]( https://peps.python.org/pep-0639/ ) defined the spec for handling license files. So we use that here